scala.html.markdown

```diff
@@ -6,6 +6,7 @@ contributors:
     - ["Dominic Bou-Samra", "http://dbousamra.github.com"]
     - ["Geoff Liu", "http://geoffliu.me"]
     - ["Ha-Duong Nguyen", "http://reference-error.org"]
+	- ["Christian Albrecht", "https://github.com/coastalchief"]
```

Comment:
Can you fix the indentation here, please? This will break the site generator I think.